docs/source/quickstart/index.rst

```diff
@@ -30,10 +30,10 @@ You probably noticed that we used ``/bin/true`` as a test, and in accordance
 with our expectations, it passed! These are known as `simple tests`, but there
 is also another type of test, which we call `instrumented tests`.
 
-.. tip:: See more at the "Test types" sections to understand Avocado's test types.
+.. tip:: See more at the "Test types" section on the Avocado User's Guide.
 
-Why should I use?
-=================
+Why should I use it?
```

Comment:
Now that you've fixed it, I'm asking myself how I was reading it without catching it.